.changeset/config.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://unpkg.com/@changesets/config@1.6.0/schema.json",
-  "changelog": "@changesets/cli/changelog",
+  "changelog": "@changesets/changelog-github",
```

Comment:
This updates our CHANGELOG generator to include links to the commit SHA, PR links, and a thank you to the author.